frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPalette.test.tsx

```diff
@@ -25,12 +27,26 @@ const schema: SchemaStore = {
   },
 }
 
+const ActiveTableNameDisplay: FC = () => {
+  const userEditing = useContext(UserEditingContext)
+
+  return (
+    // The currently active table name is displayed via Context. This component is used in tests for assertions only.
+    <div data-testid="test-active-table-name-display">
+      {userEditing?.activeTableName}
+    </div>
+  )
+}
+
 const wrapper = ({ children }: { children: ReactNode }) => (
-  <NuqsAdapter>
-    <UserEditingProvider>
-      <SchemaProvider schema={schema}>{children}</SchemaProvider>
-    </UserEditingProvider>
-  </NuqsAdapter>
+  <NuqsTestingAdapter>
```

Comment:
It's recommended to use `NuqsTestingAdapter` in tests instead of `NuqsAdapter`, otherwise the state is not cleared after each test case.

https://nuqs.47ng.com/docs/testing#nuqstestingadapter